magma/digital.py

```diff
@@ -127,7 +127,7 @@ def qualify(cls, direction):
         return cls[direction]
 
     def __eq__(cls, rhs):
-        return cls is rhs
+        return isinstance(rhs, DigitalMeta)
```

Comment:
these are probably almost always equivalent in our code base, but when is it preferred to do `isinstance(T, XMeta)` versus `issubclass(T, X)`?

I imagine sometimes these conditions could different, but when it's not, ideally we are consistent.

Reply:
The benefit of using isinstance is shown above, in that it can take a non-class argument and return False (whereas is 
subclass will raise a type error)